src/Tools/dotnet-dump/Commands/SOSCommand.cs

```diff
@@ -0,0 +1,90 @@
+using Microsoft.Diagnostic.Utilities;
+using SOS;
+using System.CommandLine;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostic.Tools.Dump
+{
+    [Command(Name = "sos", Help = "Executes a SOS command.")]
+    [CommandAlias(Name = "bpmd",  Expansion = "sos bpmd", Help = "Creates a breakpoint at the specified managed method in the specified module.")]
```

Comment:
Commands like bpmd don't make any sense to have for a dump. I think we should curate what we bring over.

Reply:
Yes, I missed that. I just copied all the aliases from the lldb plugin.  I think the rest of the aliases will work, but I'll have to verify to make sure.